server/serializers/subscriptions.go

```diff
@@ -176,6 +176,23 @@ type Definition struct {
 	Name string `json:"name"`
 }
 
+type ReleaseDetails struct {
+	Name              string            `json:"name"`
+	ID                int               `json:"id"`
+	Status            string            `json:"status"`
+	Environments      []Environments    `json:"environments"`
+	Link              Link              `json:"_links"`
+	ReleaseDefinition ReleaseDefinition `json:"releaseDefinition"`
+}
+
+type Environments struct {
+	Name string `json:"name"`
+}
+
+type ReleaseDefinition struct {
+	Name string `json:"name"`
+}
```

Comment:
https://github.com/Brightscout/mattermost-plugin-azure-devops/pull/104#discussion_r1018774655
same

Reply:
here although we have other structs with `name` JSON field but created this separate for code readability and tally with API response as using a generic one may create confusion to match with API response structure